apps/api/src/domain/models/digital-text/commands/add-content-to-digital-text-page/content-added-to-digital-text-page.event-handler.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { LanguageCode } from '@coscrad/api-interfaces';
+import { INestApplication } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Test } from '@nestjs/testing';
+import buildMockConfigService from '../../../../../app/config/__tests__/utilities/buildMockConfigService';
+import buildConfigFilePath from '../../../../../app/config/buildConfigFilePath';
+import { Environment } from '../../../../../app/config/constants/environment';
+import { ArangoConnectionProvider } from '../../../../../persistence/database/arango-connection.provider';
+import { ArangoDatabaseProvider } from '../../../../../persistence/database/database.provider';
+import { PersistenceModule } from '../../../../../persistence/persistence.module';
+import generateDatabaseNameForTestSuite from '../../../../../persistence/repositories/__tests__/generateDatabaseNameForTestSuite';
+import { DigitalTextViewModel } from '../../../../../queries/digital-text';
+import { buildTestInstance } from '../../../../../test-data/utilities';
+import buildDummyUuid from '../../../__tests__/utilities/buildDummyUuid';
+import { ArangoDigitalTextQueryRepository } from '../../queries/arango-digital-text-query-repository';
+import { IDigitalTextQueryRepository } from '../../queries/digital-text-query-repository.interface';
+import { ContentAddedToDigitalTextPage } from './content-added-to-digital-text-page.event';
+import { ContentAddedToDigitalTextPageEventHandler } from './content-added-to-digital-text-page.event-handler';
+
+const digitalTextId = buildDummyUuid(34);
+
+const existingText = 'content for page';
+
+const languageCodeForContent = LanguageCode.Chilcotin;
+
+const identifier = '56';
+
+const existingDigitalTextView = buildTestInstance(DigitalTextViewModel, {
```

Comment:
Need to update this object to include the `pages` array with a page that has the `identifier`.  I got it to work by adding `pages: [{ identifier: identifier }],` in the line after `id: digitalTextId,`